script/tool/README.md

```diff
@@ -1,8 +1,60 @@
 # Flutter Plugin Tools
 
+Note: The commands in tools are designed to run under plugins/ or plugins/packages/.
+
 To run the tool:
 
 ```sh
 dart pub get
 dart run lib/src/main.dart <args>
 ```
+
+## Format Code
+
+```sh
+cd <path_to_plugins>/packages
```

Comment:
Remove "packages" from all of these examples; it's not necessary to be there, and it makes the next line wrong.